shared/src/main/scala/amf/core/client/scala/errorhandling/IgnoringErrorHandler.scala

```diff
@@ -5,13 +5,14 @@ import amf.core.internal.validation.core.ValidationSpecification
 import org.mulesoft.lexer.SourceLocation
 import org.yaml.model.{SyamlException, YError}
 
+/** Error handler that ignores constraint reports */
```

Comment:
-> `that ignors errors` is better. The user doesn't know what `contraint reports` are.